src/test/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImplTestDV.java

```diff
@@ -0,0 +1,156 @@
+                actualDvDocument.getInvoiceNumber());
+    }
+
+    private VendorService buildMockVendorService() {
+        VendorService vendorService = Mockito.mock(VendorService.class);
+        DocGenVendorFixture[] vendorFixtures = { DocGenVendorFixture.XYZ_INDUSTRIES, DocGenVendorFixture.REE_PHUND };
+        for (DocGenVendorFixture vendorFixture : vendorFixtures) {
+            Mockito.when(vendorService.getByVendorNumber(vendorFixture.getVendorNumber()))
+                    .thenReturn(vendorFixture.createVendorDetail());
+        }
+        return vendorService;
+    }
+
+    private CuDisbursementVoucherDefaultDueDateService buildCuDisbursementVoucherDefaultDueDateService() {
+        CuDisbursementVoucherDefaultDueDateService service = Mockito
+                .mock(CuDisbursementVoucherDefaultDueDateService.class);
+        Calendar calendar = Calendar.getInstance();
+        calendar.add(Calendar.DAY_OF_MONTH, 1);
+        Mockito.when(service.findDefaultDueDate()).thenReturn(new java.sql.Date(calendar.getTimeInMillis()));
+        return service;
+    }
+
+    private CuDisbursementVoucherPayeeService buildCuDisbursementVoucherPayeeService() {
+        return new CuDisbursementVoucherPayeeServiceImpl();
+    }
+
+    protected static class DvTestCreateAccountingDocumentServiceImpl extends TestCreateAccountingDocumentServiceImpl {
+
+        public DvTestCreateAccountingDocumentServiceImpl(PersonService personService,
+                AccountingXmlDocumentDownloadAttachmentService downloadAttachmentService,
+                ConfigurationService configurationService, UniversityDateService universityDateService,
+                DateTimeService dateTimeService,
+                CuDisbursementVoucherDefaultDueDateService cuDisbursementVoucherDefaultDueDateService,
+                CuDisbursementVoucherPayeeService cuDisbursementVoucherPayeeService, VendorService vendorService) {
+            super(personService, downloadAttachmentService, configurationService, universityDateService,
+                    dateTimeService);
+            this.cuDisbursementVoucherDefaultDueDateService = cuDisbursementVoucherDefaultDueDateService;
+            this.cuDisbursementVoucherPayeeService = cuDisbursementVoucherPayeeService;
+            this.vendorService = vendorService;
+        }
+
+        @Override
+        protected AccountingDocumentGenerator<?> buildAccountingDocumentGenerator(
+                BiFunction<Supplier<Note>, Supplier<AdHocRoutePerson>, AccountingDocumentGeneratorBase<?>> generatorConstructor) {
+            CuDisbursementVoucherDocumentGenerator dvGenerator = (CuDisbursementVoucherDocumentGenerator) super.buildAccountingDocumentGenerator(
+                    generatorConstructor);
+            dvGenerator.setUniversityDateService(universityDateService);
+            dvGenerator.setDisbursementVoucherTravelService(disbursementVoucherTravelService);
```

Comment:
Can the DisbursementVoucherTravelService variable be removed from the CuDisbursementVoucherGenerator class?